libsolidity/parsing/Parser.cpp

```diff
@@ -134,6 +134,10 @@ ASTPointer<SourceUnit> Parser::parse(CharStream& _charStream)
 					nodes.push_back(parseVariableDeclaration(options));
 					expectToken(Token::Semicolon);
 				}
+				else if (
+					m_scanner->currentToken() == Token::Type
+				)
```

Comment:
```suggestion
				else if (m_scanner->currentToken() == Token::Type)
```

Reply:
Actually why not move this to a `case`? The part here should only be the fallback / workaround case